research/src/pages/charter.research.mdx

```diff
@@ -0,0 +1,113 @@
+---
+name: Charter
+path: /charter
+---
+
+Most recent revision: April 1, 2020
+
+# Vision
+
+The web deserves interoperability between component frameworks and design systems.
+The UI community is in a similar place as the industrial revolution prior to standardization.
+Untold amounts of human effort are exhausted globally every day on rebuilding identical components for web apps.
+
+This should not be the case.
+The UI community should standardize namings and structures for common components.
+We should also standardize a way of theming these components.
+We should set a path for existing solutions to converge and for browsers to natively provide these things in the future.
+
+UI component patterns have evolved and stabilized but have not made their way to to browsers or standards.
+Designers and developers reinvent the same components for every product they build.
+When building a web app or web page designers and developers should have a common set of components at their disposal.
+We shouldn't have to rebuild a dropdown, modal dialog, split button, or other components before we build our products.
+
+# Goals
+
+These are the goals Open UI believes will realize the above vision.
+
+- Document component names as they exist today
+- A common language for describing UIs and design systems
+- Eventual browser standards for web app components
+- Converging designer processes and developer workflows
+
+# Scope of work
+
+- Utilize descriptivism of common patterns to identify and standardize on:
+  - Standardize component names and parts
+  - Standardize states
+  - Standardize on inputs that transition states (eg: keyboard or pointer interaction)
+- Cover for each component the necessary accessibilty requirements (this is not in place of
+  WIA-ARIA or WICAG but linking where appropriate to alleviate implementor pain)
+- Upon discovery of gaps in the web platform, denote those and raise relevant issues to
+  standards bodies
+
+# Out of scope
+
+- Standardization of default appearance of the component
+
+# Process for creation of a pull request
+
+The below process aims for efficiency and asynchronous changes but also allows for transparency
+and opportunities to ensure consesus from the community.
+
+- Research should be done across the industry for each aspect of the component
+- Based on investigation, curate decisions and propose an Open UI resolution
+- Open a PR regarding your proposal
+- Two members of the community group need to review and sign off on the change\
+- Merge of the review sign off will be done by the editors of Open UI. Current editors are:
+  - Greg Whitworth
+  - Levi Thomason
```

Comment:
Should we say "Current editors are listed in the CODEOWNERS file"?  This way it stays fresh.